sample/lit-todo/index.html

```diff
@@ -68,93 +94,73 @@
             await tx.put(key(todo.id), todo);
         });
 
-        // TODO: Remove these by auto-syncing after mutate.
-        const handleCheckbox = async (id, e) => {
-            await updateTodo({ id, complete: e.srcElement.selected });
-            rep.sync();
-        };
-
-        const handleDelete = async (id, e) => {
-            cancelEvent(e);
-            await deleteTodo({ id });
-            rep.sync();
-        };
-
-        const cancelEvent = e => {
-            e.stopPropagation();
-        }
-
-        const handleCreate = async () => {
-            const first = (await rep.query(tx => toArray(tx.scan({ prefix: '/todo/', limit: 1 }))))[0];
+        async function handleCreate() {
+            const [first] = todos;
             await createTodo({
                 id: Math.round(Math.random() * Math.pow(2, 30)),
                 listID: 2,
                 text: 'Untitled',
                 order: first ? (first.order / 2) : 0.5,
                 complete: false,
             });
-            document.querySelector('mwc-check-list-item .text').focus();
-            rep.sync();
+            document.querySelector('.todo-list .textwrap span').focus();
         };
 
-        const handleFocus = e => {
+        async function handleDelete(id, e) {
+            e.stopPropagation();
```

Comment:
Did you mean preventDefault?

Reply:
I don't think so - I need to stop the event from going up to the todo item because at that level it is interpreted to check the checkbox.

Reply:
Makes sense. I prefer comments on these since it is too common for people to blindly just stop the propagation

Reply:
Comment incoming in next review.